docs/navigation-configuration.md

```diff
@@ -286,7 +286,7 @@ When loading, the **viewUrl** uses the following dynamic URL parameters:
 - `sort = asc`
 
 ```
-Luigi.setConfig({
+window.Luigi.setConfig({
```

Comment:
It still should work with just `Luigi.setConfig` right?

Reply:
yes, everywhere is used with Luigi.setConfig. I just added it here to the documentation for consistency

Reply:
So it's ok with this task, but I guess for the future we should set it to just `Luigi.setConfig`, it just seems easier ;)